freqtrade/data/entryexitanalysis.py

```diff
@@ -148,11 +157,23 @@ def _do_group_table_output(bigdf, glist):
                 new['mean_profit_pct'] = new['mean_profit_pct'] * 100
                 new['total_profit_pct'] = new['total_profit_pct'] * 100
 
-                _print_table(new, sortcols)
+                _print_table(new, sortcols, name=f"Group {g}:",
+                             to_csv=to_csv, csv_path=csv_path)
             else:
                 logger.warning("Invalid group mask specified.")
 
 
+def _do_rejected_signals_output(rejected_signals_df, to_csv=False, csv_path=None):
+    cols = ['pair', 'date', 'enter_tag']
+    sortcols = ['date', 'pair', 'enter_tag']
+    _print_table(rejected_signals_df[cols],
```

Comment:
I find the output quite odd in case there's no rejected signal.
i've had to tripple-check to see why i get no output before realizing that i probably had no rejected signals in my backtest.

maybe adding an output "someplace" which says "no rejected signals" would make this userfriendlier?

Reply:
Sure, I can add this.